CanvasPlusPlayground/Features/Courses/CourseListView.swift

```diff
@@ -139,12 +139,23 @@ struct CourseListView: View {
                     showAuthorization.toggle()
                 }
             }
-
+            
+#if DEBUG
```

Comment:
Can you remove these flag addition changes? I have an issue (https://github.com/gtiosclub/CanvasPlusPlayground/issues/83) in which I'd like to incorporate some more changes.